ads/google/imaVideo.js

```diff
@@ -322,6 +321,8 @@ export function imaVideo(global, data) {
   setStyle(videoPlayer, 'background-color', 'black');
   videoPlayer.setAttribute('poster', data.poster);
   videoPlayer.setAttribute('playsinline', true);
+  videoPlayer.setAttribute(
```

Comment:
Not clear what this accomplishes given we use custom controls. (also controlsList is Chrome only)